flareon/src/db.rs

```diff
@@ -821,8 +912,7 @@ pub enum ColumnType {
     Time,
     Date,
     DateTime,
-    Timestamp,
-    TimestampWithTimeZone,
+    DateTimeWithTimeZone,
```

Comment:
Why?

Reply:
There seems to be no consensus on what does "datetime"/"timestamp" mean in database engines:
* SQLite doesn't have a concept of a date(time), you just store it as text
* Postgres doesn't have a dedicated datetime field; instead it just has timestamp with a wide range of possible values
* MySQL has both: datetime is anything between year 1000 and 9999, while timestamp is 32-bit unix timestamp, so it's susceptible to Y2038 problem. MariaDB is slightly better in this regard, but it only supports timestamps with years up to 2106.

I thought it might be surprising for users who might want to store arbitrary `chrono::DateTime` objects and see them failing if they don't fall between 1970 and 2038. So I changed the naming so that it's clear it's not a timestamp, and we might eventually add a separate "timestamp" type in the future. For now, I just want to make sure that whatever (sensible) value the users might want to store, will work.